src/renderer/api/endpoints/helm-charts.api.ts

```diff
@@ -86,7 +86,7 @@ export class HelmChart {
   tillerVersion?: string;
 
   getId() {
-    return this.digest;
+    return `${this.apiVersion}/${this.name}@${this.getAppVersion()}`;
```

Comment:
just curious, what is the motivation of this change? is `this.digest` broken somehow?

Reply:
Yes, it's the same for all charts (at least was some days ago)

Reply:
Let's split this into separate PR (and/or issue).

Reply:
Ok